### PR TITLE
Book request

### PR DESCRIPTION
**Pull request by isabloCh**
changed the functionality of obtaining information about the book.

### CODE REVIEW

File: app/javascript/vue/books-data.js [hunk 1]
```diff
@@ -11,15 +10,8 @@ function getAll () {
...
-      .then((books) => {
-        const book = _.find(books, { id: id })
-        if (book) {
-          resolve(book)
-        } else {
-          reject(new Error(`Can't find book with ID ${id}`))
-        }
-      })
+    axios(`/api/books/${id}`)
+      .then((response) => resolve(response.data))
```
Comment (resivalex) [inline app/javascript/vue/books-data.js]: Does it work? Don't you need `response.data.book`?

**Review by resivalex — CHANGES_REQUESTED** [review]

File: app/javascript/vue/books-data.js (hunk 1 above)
Comment (isabloCh) [reply app/javascript/vue/books-data.js]: works. but how else?

Comment (resivalex) [reply app/javascript/vue/books-data.js]: I think you won't get a book, you'll get an object with the `book` key

**Review by resivalex — APPROVED** [review]

File: app/controllers/api/books_controller.rb [hunk 2]
```diff
@@ -8,5 +8,9 @@ def index
       end)
     end
 
+    def show
+      render json: { book: Book.find(params[:id]) }
+    end
+
```
Comment (Loremaster) [inline app/controllers/api/books_controller.rb:14]: @resivalex I am very surprised that rubocop doesn't complain here. It should tell `Layout/TrailingWhitespace: Trailing whitespace detected.`. Do you know why is it so?

Comment (resivalex) [reply app/controllers/api/books_controller.rb:14]:
```
Layout/EmptyLinesAroundClassBody:
  EnforcedStyle: empty_lines_except_namespace
```
Look at this one, please

Comment (isabloCh) [reply app/controllers/api/books_controller.rb:14]: Gods what's going on

Comment (Loremaster) [reply app/controllers/api/books_controller.rb:14]: @resivalex ah I see thanks. I think that default option (`no_empty_lines`) produces a bit more cleaner code but it's just a personal taste. Let's leave it like that for now.

Comment (Loremaster) [reply app/controllers/api/books_controller.rb:14]: It's your lucky day @isabloCh haha!

**Review by Loremaster — APPROVED** [review]

File: app/controllers/api/books_controller.rb (hunk 2 above)
Comment (resivalex) [reply app/controllers/api/books_controller.rb:14]: 🤣

**Pull request merged**